package.json

```diff
@@ -85,4 +88,4 @@
   },
   "stability": "stable",
   "types": "src/index.d.ts"
-}
+}
```

Comment:
missing newline

Reply:
@narekhovhannisyan in cases like this, please just add the newline and commit. It's silly for these changes to involve >1 person.

Reply:
```suggestion
}

```

Reply:
```suggestion
}

```